base-notebook/Dockerfile

```diff
@@ -51,10 +51,9 @@ RUN apt-get -q update \
     sudo \
     locales \
     fonts-liberation \
-    run-one \
- && apt-get clean && rm -rf /var/lib/apt/lists/*
-
-RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
+    run-one && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* && \
+    echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
```

Comment:
I think having separate steps is actually good in this case (there is a clear distinction in the processes), so I would prefer hadolint ignore for this line.

Reply:
After a closer look, it actually makes sense both ways, because several lines above we install locales. So, I'm ok with this change.

Reply:
So if you are Ok :-)